src/main/scala/temple/generate/target/openapi/OpenAPIGenerator.scala

```diff
@@ -22,6 +22,7 @@ private class OpenAPIGenerator private (name: String, version: String, descripti
   private val errorTracker = FlagMapView(
     400 -> generateError("Invalid request", "Invalid request parameters: name"),
     401 -> generateError("Valid request but forbidden by server", "Not authorised to create this object"),
+    403 -> generateError("Valid request but server will not fulfill", "User with this ID already exists"),
```

Comment:
This made me think, what if the user adds endpoints in their service, then updates the openAPI doc. It gets overwritten when regenerating 😱 do we need hooks for openAPI?!!!

Reply:
Good point.. no idea how we’d do that 😂